client/state/data-layer/wpcom-http/pipeline/retry-on-failure/index.jsx

```diff
@@ -0,0 +1,108 @@
+/**
+ * External dependencies
+ */
+import {
+	get,
+	head,
+	sortBy,
+	toPairs,
+} from 'lodash';
+
+const requestQueue = new Map();
```

Comment:
> I'm realizing now that this would never actually make it to Calypso right?

if it's stored in an action it will and we should be able to see it in the action log

you raise some good points.